dist/main.css

```diff
@@ -1,4 +1,32 @@
+* {
+  margin: 0; }
+
+html {
+  box-sizing: border-box; }
+
+header {
+  background-color: #313030; }
+
 body {
-  padding: 0;
-  margin: 0;
-  border: 1px solid; }
+  font-family: 'Open Sans', sans-serif; }
+
+li {
+  list-style-type: none; }
+
+.nav-list-item {
+  margin: 0 58px 0 58px;
+  padding: 34px 0 34px 0; }
+
+li a:hover {
```

Comment:
the module must have only classes as a base. The `li` element is your `.nav-list-item` , the same for `a` selector, it is your `.nav-list-link`